controllers/helmrelease_controller.go

```diff
@@ -131,6 +134,9 @@ func (r *HelmReleaseReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error)
 		return ctrl.Result{Requeue: true}, updateStatusErr
 	}
 
+	// Record ready status
+	r.recordReadiness(hr, false)
```

Comment:
We should also call this on L165 and L322 (after the calls to `v2.HelmReleaseProgressing()`)

Reply:
Done!